draft-ietf-jsonpath-base.md

```diff
@@ -1408,15 +1408,15 @@ is a subset of the set of instances of the other type.
 {{tbl-types}} defines the available types in terms of abstract instances, where `n` denotes a node, `v` denotes a value, and `nl` denotes
 a non-empty nodelist. The table also lists the subtypes of each type.
 
-| Type                  | Abstract Instances                       | Subtypes                                 |
-| :--:                  | :----------------:                       | :------:                                 |
-| `OptionalNodeOrValue` | `Node(n)`, `Value(v)`, `Nothing`         | `OptionalNode`, `OptionalValue`, `Value` |
-| `OptionalNode`        | `Node(n)`, `Nothing`                     |                                          |
-| `OptionalValue`       | `Value(v)`, `Nothing`                    | `Value`, `OptionalBoolean`               |
-| `Value`               | `Value(v)`                               | `Boolean`                                |
-| `OptionalBoolean`     | `Value(true)`, `Value(false)`, `Nothing` | `Boolean`                                |
-| `Boolean`             | `Value(true)`, `Value(false)`            |                                          |
-| `OptionalNodes`       | `Nodes(nl)`, `Nothing`                   | `OptionalNode`                           |
+| Type                  | Abstract Instances                       | Subtypes                                                               |
+| :--:                  | :----------------:                       | :------:                                                               |
+| `OptionalNodeOrValue` | `Node(n)`, `Value(v)`, `Nothing`         | `OptionalNode`, `OptionalValue`, `OptionalBoolean`, `Value`, `Boolean` |
+| `OptionalNode`        | `Node(n)`, `Nothing`                     |                                                                        |
+| `OptionalValue`       | `Value(v)`, `Nothing`                    | `OptionalNode`, `Value`, `OptionalBoolean`, `Boolean`                  |
```

Comment:
see https://github.com/ietf-wg-jsonpath/draft-ietf-jsonpath-base/issues/385#issuecomment-1420298763

Reply:
@cabo Please can we treat the current PR as editorial, i.e. making the table consistent by including the subtypes by coercion. Then we can merge it, @gregsdennis can do the type renaming ("XxxType"), and we can work on subtypes by coercion to see what better alternatives there are in the context of the four recent issues.

Reply:
Sure